src/updater/main.tsx

```
@@ -0,0 +1,14 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import '@/i18n'
+import { initializeWindowUi } from '@/lib/window-ui'
+import '@/styles/globals.css'
+import UpdaterWindow from './UpdaterWindow'
```

ESLint

[error] `./UpdaterWindow` import should occur before import of `@/lib/window-ui`

(import-x/order)